src/test/java/org/gridsuite/modification/server/utils/elasticsearch/DisableElasticsearch.java

```diff
@@ -23,5 +24,6 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @MockBeans({@MockBean(EmbeddedElasticsearch.class), @MockBean(EquipmentInfosRepository.class), @MockBean(TombstonedEquipmentInfosRepository.class)})
+@Tag("Docker")
```

Comment:
Why did you add tags, is it just for informational purposes or it's planned to be used in the CI/CD ?

Reply:
It's to distinct units tests from integration tests, as they aren't separated in folders (like `tests\, ` it-tests`). It also permit to run for example unit tests quickly.

Reply:
Ok, I used tags too before to optimise CI/CD runtime, but here it's not used so I was wondering the purpose